### ISSUE TITLE
Cursor displayed as 'pointer' when background closing is disabled

### ISSUE THREAD

**Issue by BWPanda** [noelboss/featherlight#325]
When you disable the closing of the lightbox by clicking on the background with `closeOnClick: false`, the cursor is still shown as a 'pointer' when hovering over the background. This can make it confusing for users who think they can click something when they cannot.

See example here: http://jsfiddle.net/JNsu6/449/

I'd recommend making the cursor 'default' when `closeOnClick` is `false`.

**Comment by marcandre** [comment]
Agreed. Care to try a PR? Maybe best approach is to add take care of a particular class (say `featherlight-close-with-click`) on the overlay? I'm not sure how hard or easy it will be to support nested featherlights with different settings.

**Comment by BWPanda** [comment]
Sorry, I was writing a Backdrop CMS module for this plugin when I noticed this (and my other) issue. I'm not much of a JS coder, so will leave this for those that are. Just wanted to mention these bugs in case you weren't already aware.
